src/main/kotlin/io/github/anki/anki/exception/ControllerExceptionHandler.kt

```diff
@@ -0,0 +1,40 @@
+package io.github.anki.anki.exception
```

Comment:
this shouldn't be located at exception package, move to controller

Reply:
+